website/photos/forms.py

```diff
@@ -1,10 +1,37 @@
 from django import forms
+from django.core.files.base import File
 from django.utils.translation import gettext_lazy as _
 
+from django_drf_filepond.models import TemporaryUpload
+
 from photos.models import Album, Photo
 from photos.validators import ArchiveFileTypeValidator
 
 
+class FilePondWidget(forms.FileInput):
+    needs_multipart_form = True
+    template_name = "photos/fotopondwidget.html"
+
+    def value_from_datadict(self, data, files, name):
+        upload_id = data.get(name)
+        if upload_id is None:
+            return files.get(name)
+        return upload_id
+
+    def value_omitted_from_data(self, data, files, name):
+        return name not in data and name not in files
+
+
+class FilePondField(forms.FileField):
+    widget = FilePondWidget
+
+    def to_python(self, data):
+        if isinstance(data, str):
+            temp_upload = TemporaryUpload.objects.get(upload_id=data)
+            return File(temp_upload.file, name=temp_upload.upload_name)
+        return super().to_python(data)
+
+
```

Comment:
Maybe move this to utils or a completely separate `django-filepond-widget` package